files/startup-script-stdlib.sh

```diff
@@ -0,0 +1,230 @@
+#! /bin/bash
+#
+# Copyright 2018 Google LLC. This software is provided as-is,
+# without warranty or representation for any use or purpose.
+# Your use of it is subject to your agreements with Google.
+
+# Standard library of functions useful for startup scripts.
+
+# Pending UX behaviors
+# TODO(jmccune): load_config_values to load user configuration
+# TODO(jmccune): call mandatory_argument() before E_MISSING_MANDATORY_ARG
+# TODO(jmccune): add -c <name> for alternate startup-script-custom name
+# Pending initialization functions:
+# TODO(jmccune): gsutil initialization w/ crcmod
+# Pending operational functions:
+# TODO(jmccune): get_from_bucket()
+# TODO(jmccune): setup_init_script()
+# TODO(jmccune): setup_sudoers()
```

Comment:
This file contains the library of startup script functions and the TODO items are what will be added in subsequent pull requests which have already been used in the field.